projects/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -42,16 +41,10 @@
  * @since 4.3.0
  */
 class Jetpack_Core_Json_Api_Endpoints {
-
 	/**
-	 * @var string Generic error message when user is not allowed to perform an action.
+	 * Roles that can access Stats once they're granted access.
 	 *
-	 * @deprecated 8.8.0 Use `REST_Connector::get_user_permissions_error_msg()` instead.
-	 */
-	public static $user_permissions_error_msg;
```

Comment:
Looks like there's still a reference to this at
https://github.com/Automattic/jetpack/blob/d354c88ff284fa9d5fe13f90733358817f0ebca8/projects/plugins/jetpack/_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php#L607

Reply:
Good catch! 65ba8e0d040f60694c29872e85c94f969755aca5